java/templates/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -221,6 +221,9 @@ public class ApiClient {
 
         // Set default User-Agent.
         setUserAgent("{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}OpenAPI-Generator/{{{artifactVersion}}}/java{{/httpUserAgent}}");
+        Random randomGenerator = new Random();
+        long randomId = randomGenerator.nextLong();
+        addDefaultHeader("svix-req-id", String.valueOf(Math.abs(randomId)));
```

Comment:
Why do you need abs?

Reply:
It can be a negative number otherwise, maybe that's fine?

Reply:
Ahhh, sorry. My bad, I thought it was round.